libs/s25main/ai/AIInterface.cpp

```diff
@@ -340,3 +343,15 @@ bool AIInterface::CallSpecialist(const noFlag* flag, Job job)
 {
     return CallSpecialist(flag->GetPos(), job);
 }
+
+void AIInterface::Chat(const std::string& message)
```

Comment:
Can we add the chat destination here too, otherwise I'll still need to use the old/existing interface for team communication.

Reply:
Good point. Done